pkg/node/local_node_store.go

```diff
@@ -116,6 +127,7 @@ func NewLocalNodeStore(params LocalNodeStoreParams) (*LocalNodeStore, error) {
 			s.emit = emit
```

Comment:
given we protect these fields with the `getMu` in the stop hook, why don't we need to protect them here?